codecept.conf.js

```diff
@@ -9,8 +9,9 @@ module.exports = {
   name: 'right-to-rent-check',
   tests: './apps/**/acceptance/features/*.js',
   include: {
+    tenantInUKPage: pagesPath('tenant-in-UK.js'),
```

Comment:
Please don't use mixed-case filenames. Windows will forgive you if you try to load that file with all lowercase but linux will not.